pkg/maps/nat/nat.go

```diff
@@ -120,7 +120,7 @@ func NewMap(name string, v4 bool, entries int) *Map {
 }
 
 func (m *Map) Delete(k bpf.MapKey) error {
-	return (&m.Map).Delete(k)
+	return (&m.Map).SilentDelete(k)
```

Comment:
This method is invoked in https://github.com/cilium/cilium/blob/master/pkg/maps/ctmap/ctmap.go#L615. I think for this case we might need to make `SilentDelete()` to return a bool in addition to indicate whether an entry has been removed. Otherwise, the metric counters in the linked code might get wrongly incremented.